javascript/javascript2/week3/gamepresser.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="game.css" />
+    <title>fastest presser</title>
+</head>
+<body>
+    <header>
+        <h2>Fastest Presser</h2>
+    </header>
+
+    <div class="inputtag"><input type="number" id="number" name="game" placeholder="Set Seconds" min="0"><br><br></div>
+   
+    <div id="startbutton"><button id="button">Start game!</button></div>
+    <div id="time">
+    <P id="count_down"></P>
+    <h2 id="timeupmessage"></h2>
+</div>
+    <hr>
+    <section>
+    <div class="scount">
+        <h1>Press S</h1>
+        <h4 id="count_s"></h4>
```

Comment:
```suggestion
        <h4 id="count-s"></h4>
```

So you now have a count-s and a s-count. Which does what? Its very hard to know. Give descriptive names